src/js/movies.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @author Jörgen Lindström
+ */
+//console.log('movies.js loaded');
+
+let moviesArray = [];
+
+export const fetchMovieData = async () => {
+    try {
+        const response = await fetch('../../public/database/movies.json');
+        moviesArray = await response.json(); // Populate the array
+        console.log(moviesArray);
+    } catch (error) {
+        console.error('Error fetching data:', error);
```

Comment:
I recommend to  add a fallback value incase of an error,
``` return null; ```

Reply:
Where do you recommend to write that in the code?